Anlab.Mvc/Extensions/StringExtensions.cs

```diff
@@ -57,24 +57,33 @@ public static bool IsEmailValid(this string value)
 
         public static string FormatPhone(this string value)
         {
-            if (string.IsNullOrWhiteSpace(value))
-            {
-                return value;
-            }
-            var phoneNumberUtil = PhoneNumbers.PhoneNumberUtil.GetInstance();
-            var phoneNumber = phoneNumberUtil.ParseAndKeepRawInput(value, "US");
-            var phone = phoneNumberUtil.Format(phoneNumber, PhoneNumberFormat.NATIONAL);
-            if (!phoneNumberUtil.IsValidNumberForRegion(phoneNumber, "US"))
+            try
             {
-                if (phoneNumberUtil.IsValidNumber(phoneNumber))
+                if (string.IsNullOrWhiteSpace(value))

```

Comment:
@jSylvestre no point in putting this in the try/catch.  But i guess not a problem either